WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -301,11 +301,19 @@ private void refreshBlogDetails() {
         mBlavatarImageView.setImageUrl(GravatarUtils.blavatarFromUrl(mBlog.getUrl(), mBlavatarSz), WPNetworkImageView.ImageType.BLAVATAR);
 
         String blogName = StringUtils.unescapeHTML(mBlog.getBlogName());
-        String hostName = StringUtils.getHost(mBlog.getUrl());
-        String blogTitle = TextUtils.isEmpty(blogName) ? hostName : blogName;
+        String homeURL;
+        if (!TextUtils.isEmpty(mBlog.getHomeURL())) {
+            homeURL = mBlog.getHomeURL().replace("http://", "").replace("https://", "").trim();
```

Comment:
This same code is used [here](https://github.com/wordpress-mobile/WordPress-Android/pull/3186/files#diff-76943ef3860089db56505865981128cfR39). Rather than duplicate the code like this, it would better to make it a separate routine. In this case, two routines would be even better since we'll likely use them again: a StringUtils method to strip the trailing slash, and a UrlUtils method to remove `http://` and `https://`.